tests/pr_test.go

```diff
@@ -2,84 +2,88 @@
 )
 
 // Use existing resource group
-const basicExampleDir = "examples/basic"
-const completeExampleDir = "examples/complete"
+const rootDaDir = "./"
 
 // Current supported regions
 var validRegions = []string{
 	"us-south",
 	"eu-de",
 }
 
-func setupOptionsBasicExample(t *testing.T, prefix string, dir string) *testhelper.TestOptions {
-	options := testhelper.TestOptionsDefault(&testhelper.TestOptions{
-		Testing:      t,
-		TerraformDir: dir,
-		IgnoreDestroys: testhelper.Exemptions{ // Ignore for consistency check
-			List: []string{
-				"module.watsonx_saas.module.configure_user.null_resource.configure_user",
-				"module.watsonx_saas.module.configure_user.null_resource.restrict_access",
-			},
-		},
-		IgnoreUpdates: testhelper.Exemptions{ // Ignore for consistency check
-			List: []string{
-				"module.watsonx_saas.module.configure_user.null_resource.configure_user",
-				"module.watsonx_saas.module.configure_user.null_resource.restrict_access",
-			},
-		},
-	})
-	terraformVars := map[string]interface{}{
-		"location": validRegions[rand.Intn(len(validRegions))],
+// Define a struct with fields that match the structure of the YAML data
+const yamlLocation = "../common-dev-assets/common-go-assets/common-permanent-resources.yaml"
+
+var permanentResources map[string]interface{}
+
+// TestMain will be run before any parallel tests, used to read data from yaml for use with tests
+func TestMain(m *testing.M) {
+
+	var err error
+	permanentResources, err = common.LoadMapFromYaml(yamlLocation)
+	if err != nil {
+		log.Fatal(err)
 	}
-	options.TerraformVars = terraformVars
 
-	return options
+	os.Exit(m.Run())
 }
 
-func setupOptionsCompleteExample(t *testing.T, prefix string, dir string) *testhelper.TestOptions {
+func setupOptionsRootDA(t *testing.T, prefix string, dir string) *testhelper.TestOptions {
 	options := testhelper.TestOptionsDefault(&testhelper.TestOptions{
 		Testing:      t,
 		TerraformDir: dir,
+		Prefix:       prefix,
 		IgnoreDestroys: testhelper.Exemptions{ // Ignore for consistency check
 			List: []string{
-				"module.watsonx_saas.module.configure_user.null_resource.configure_user",
-				"module.watsonx_saas.module.configure_user.null_resource.restrict_access",
+				"module.configure_user.null_resource.configure_user",
+				"module.configure_user.null_resource.restrict_access",
 			},
 		},
 		IgnoreUpdates: testhelper.Exemptions{ // Ignore for consistency check
 			List: []string{
-				"module.watsonx_saas.module.configure_user.null_resource.configure_user",
-				"module.watsonx_saas.module.configure_user.null_resource.restrict_access",
+				"module.configure_user.null_resource.configure_user",
+				"module.configure_user.null_resource.restrict_access",
 			},
 		},
+		TerraformVars: map[string]interface{}{
+			"location":            validRegions[rand.Intn(len(validRegions))],
+			"resource_group_name": prefix,
+			"cos_kms_crn":         permanentResources["hpcs_south_crn"],
```

Comment:
@Aayush-Abhyarthi WatsonX does not support HPCS. You are going to have to create test resource folder and deploy key protect so it can be used in this test.